phpstan.neon

```diff
@@ -0,0 +1,3 @@
+includes:
```

Comment:
this file should have .dist extension (phpstan.neon.dist)

same as for phpunit or phpcs configuration

`phpstan.neon` should be added to `.gitignore`